pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -188,6 +189,23 @@ static __always_inline int handle_retransmit(struct sock *sk, int segs) {
     return 0;
 }
 
+static __always_inline int handle_retransmit_retrans_out(struct sock *sk) {
+    conn_tuple_t t = {};
+    u64 zero = 0;
+
+    if (!read_conn_tuple(&t, sk, zero, CONN_TYPE_TCP)) {
+        return 0;
+    }
+
+    u32 retrans_out;
+    bpf_probe_read(&retrans_out, sizeof(retrans_out), &(tcp_sk(sk)->retrans_out));
```

Comment:
Is `retrans_out` a monotonic counter? If so, this code will not work as intended.

Reply:
can you help me understand why that is true? I am still pretty new to ebpf, C, etc

Reply:
Its an absolute count, not a delta. We assume it is a delta in `update_tcp_stats`. You can rectify this by passing in an extra arg to `update_tcp_stats` to specify whether the retransmit is a delta or a count.

Reply:
makes sense... will do